File: src/main/java/no/nav/pensjon/regler/domain/kode/VilkarOppfyltUTCti.java
package no.nav.pensjon.regler.domain.kode;

/**
 * Beskriver hvorvidt saksbehandler mener vilkår er oppfylt.
 */
public class VilkarOppfyltUTCti extends TypeCti {

    private static final long serialVersionUID = -2622522900456746970L;

    public VilkarOppfyltUTCti() {
        super();
    }

    public VilkarOppfyltUTCti(String kode) {
        super(kode);
    }

    public VilkarOppfyltUTCti(VilkarOppfyltUTCti vilkarOppfyltUTCti) {
        super(vilkarOppfyltUTCti);
    }
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/kode/VilkarOppfyltUTCti.kt
package no.nav.pensjon.regler.domain.kode

/**
 * Beskriver hvorvidt saksbehandler mener vilkår er oppfylt.
 */
class VilkarOppfyltUTCti : TypeCti {
    constructor() : super()
    constructor(kode: String?) : super(kode)
    constructor(vilkarOppfyltUTCti: VilkarOppfyltUTCti) : super(vilkarOppfyltUTCti)
}